frontend/src/components/BoardItem.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components/macro";
+
+export default function BoardItem({difficulty}){
+    return (
+    <BoardItemWrapper>
+        <p>{difficulty.id}</p>
+    </BoardItemWrapper>
+    )
+}
+
+const BoardItemWrapper = styled.div`
+  border: 1px solid var(--secondary-color);
+  border-radius: 10px;
+  box-shadow: 0 0 8px 3px #2f3235;
+  margin-top: 8px;
+  margin-bottom: 8px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: var(--secondary-color);
+`
```

Comment:
```suggestion
import styled from "styled-components/macro";

export default styled.div`
  border: 1px solid var(--secondary-color);
  border-radius: 10px;
  box-shadow: 0 0 8px 3px #2f3235;
  margin-top: 8px;
  margin-bottom: 8px;
  padding: 20px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  background: var(--secondary-color);
`
```

Reply:
need this for later, to make the BoardItem clickable ;)